### PR TITLE
MARKET 6513 Fix and Update Sample Hospitality App

### PR DESCRIPTION
**Pull request by cs185388**
- Removed empty fields from menu items where data doesn't exist.
- Fixed images, links and menu items.
- Fixed formatting.

NOTE: Some of the work was done through the Menu Maker web app

### CODE REVIEW

File: code/BurgersUnlimited/settings.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '+=ey4af%pw@67a!knl6t44e)$ms=6uq7x)btzilz4yl$2c7!2v'
+SECRET_KEY = 'NqYY0VNSYe7IOLoYD4Aa1WBIi*Gj4WATPl0y4Hes2tTWMmqfwB'
```
Comment (av185144) [inline code/BurgersUnlimited/settings.py:25]: which environment is this?

**Review by av185144 — CHANGES_REQUESTED** [review]

File: code/BurgersUnlimited/settings.py (hunk 1 above)
Comment (cs185388) [reply code/BurgersUnlimited/settings.py:25]: I believe it's production, should we hide this?

Comment (kk185256) [reply code/BurgersUnlimited/settings.py:25]: oh wow you are right amoli this is public repo.. this is crazy

Comment (cs185388) [reply code/BurgersUnlimited/settings.py:25]: Yeah it was showing before I picked it up, I just updated it with the current one
But I agree with both of you and will remove it

**Review by jg185352 — APPROVED** [review]

**Pull request merged**